scripts/scil_compute_density_map_from_streamlines.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""
+Compute a density map from a streamlines file.
+
+A specific value can be assigned instead of using the tract count.
+
+This script correctly handles compressed streamlines.
+"""
+
+from __future__ import division
+
+import argparse
+
+import numpy as np
+import nibabel as nb
+
+from scilpy.io.streamlines import load_tracts_over_grid, check_tracts_support
```

Comment:
You need to add these function as well, please talk to @frheault about the new dipy API for streamlines